src/component/Calculator.js

```diff
@@ -1,5 +1,6 @@
 import React from 'react';
 import './styles/Calculator.css';
+import calculate from '../logic/calculate';
```

Comment:
✅ The Calculator component should import the `calculate.js` file from the logic directory 👍🏻